### ISSUE TITLE
New helper for custom fields 

### ISSUE THREAD

**Issue by bc-zoharmuzafi** [bigcommerce/paper-handlebars#33]
We would like to implement a feature to return custom field. All custom fields returned in array, and the new feature should return the custom field from this array by its name.

**Comment by scottxtensive** [comment]
Any updates on this? Would be really handy building out product pages when using custom fields.

**Comment by cheve-ruben** [comment]
Is this any different than filtering the customfields by their name?

eg.
```
{{#filter product.custom_fields 'Colour' property="name"}}
    <meta itemprop="colour" content="{{value}}">
{{/filter}}
```

**Comment by scottxtensive** [comment]
@cheve-ruben yeah, actually came across the filter function this week... works nicely!

**Comment by Tiggerito** [comment]
It would be nice for the filter to have a case-insensitive option. We don't trust people to be consistent.
